product_categ_image/models/product_category.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2018 Akretion (http://www.akretion.com)
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    image = fields.Binary(string='Image')
```

Comment:
Shouldn't this have `attachment=True` for a better DB space management?

Reply:
Probably. But this is already used in production, so maybe we could do that when migrating to v11.

Reply:
You can add a migration script right now if you want. There's a method in openupgradelib for performing the extract. Any way, at least put a TODO here.

Reply:
:+1: to put a TODO

Please, could we merge? Thanks